lib/scaleway/bootstrap/chart_values/nginx-ingress.yaml

```diff
@@ -15,8 +15,9 @@ controller:
     # https://github.com/scaleway/scaleway-cloud-controller-manager/blob/master/docs/loadbalancer-annotations.md
     annotations:
       service.beta.kubernetes.io/scw-loadbalancer-forward-port-algorithm: roundrobin
-      service.beta.kubernetes.io/scw-loadbalancer-protocol-http: false
-      service.beta.kubernetes.io/scw-loadbalancer-proxy-protocol-v1: false
-      service.beta.kubernetes.io/scw-loadbalancer-proxy-protocol-v2: false
+      service.beta.kubernetes.io/scw-loadbalancer-protocol-http: "false"
+      service.beta.kubernetes.io/scw-loadbalancer-proxy-protocol-v1: "false"
+      service.beta.kubernetes.io/scw-loadbalancer-proxy-protocol-v2: "false"
       service.beta.kubernetes.io/scw-loadbalancer-health-check-type: tcp
+      service.beta.kubernetes.io/scw-loadbalancer-use-hostname: "true"
     externalTrafficPolicy: "Local"
```

Comment:
You need to set also 
```
  publishService:
    enabled: true
```
See previous discussion/PR about it

Reply:
Done